NEWS.rst

```diff
@@ -1,6 +1,13 @@
 Version History
 ===============
 
+
+Version 0.6.0 (2020-12-01):
+* Added spatially bandlimited line source
+* Extended documentation of various functions
```

Comment:
Sure, but do we really need to mention this, especially since its a totally generic statement carrying basically no information?